documentation/CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+This document forms the basis of release notes for MSBuild.
+
+MSBuild does **not** use Semantic Versioning.
```

Comment:
It is strange to mention what the project DOES NOT follow Semantic Versioning. Why not remove this note?

Why not change this according to the [specification](http://keepachangelog.com/en/1.0.0/) (but remove the irrelevant about Semantic Versioning):

> All notable changes to this project will be documented in this file.
> 
> The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).